index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="/styles/style.css">
+    <link rel = "icon" href =  "/pict-mobile/bxs-radiation.svg" type = "image/x-icon">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="top-page">
+            <div class="ornament-top"><img src="/pict-mobile/image_geometry_1.svg" alt=""></div>
+            <div class="menu1"><img src="/pict-mobile/ic_menu.svg" alt=""></div>
```

Comment:
good job adding an alt text on the hamburger menu for great accessibility on your website 👍, in addition to this, I suggest wrapping this hamburger with a `nav` tag to make it more semantic and more accessible